scripts/makeLegacyPlots.sh

```diff
@@ -21,25 +19,29 @@ sel5=VBFclass
 sel6=DYclass
 sel7=ttHclass
 sel8=TTclass
+#sel9=VBFloose
+#sel10=baseline
 
 # CREATE DIRECTORIES FOR THE SINGLE SELECTIONS
-mkdir -p ./LegacyPlots/Legacy$year/$channel/$sel1
-mkdir -p ./LegacyPlots/Legacy$year/$channel/$sel2
-mkdir -p ./LegacyPlots/Legacy$year/$channel/$sel3
-mkdir -p ./LegacyPlots/Legacy$year/$channel/$sel4
-mkdir -p ./LegacyPlots/Legacy$year/$channel/$sel5
-mkdir -p ./LegacyPlots/Legacy$year/$channel/$sel6
-mkdir -p ./LegacyPlots/Legacy$year/$channel/$sel7
-mkdir -p ./LegacyPlots/Legacy$year/$channel/$sel8
+#mkdir -p ./LegacyPlots/Legacy$year/$channel\_$tag/$sel1
+#mkdir -p ./LegacyPlots/Legacy$year/$channel\_$tag/$sel2
+#mkdir -p ./LegacyPlots/Legacy$year/$channel\_$tag/$sel3
+#mkdir -p ./LegacyPlots/Legacy$year/$channel\_$tag/$sel4
+#mkdir -p ./LegacyPlots/Legacy$year/$channel\_$tag/$sel5
+#mkdir -p ./LegacyPlots/Legacy$year/$channel\_$tag/$sel6
+#mkdir -p ./LegacyPlots/Legacy$year/$channel\_$tag/$sel7
+#mkdir -p ./LegacyPlots/Legacy$year/$channel\_$tag/$sel8
+#mkdir -p ./LegacyPlots/Legacy$year/$channel/$sel9
+#mkdir -p ./LegacyPlots/Legacy$year/$channel/$sel10 
 # COPY THE index.php FILE
-cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel1
-cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel2
-cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel3
-cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel4
-cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel5
-cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel6
-cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel7
-cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel8
+#cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel1
+#cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel2
+#cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel3
+#cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel4
+#cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel5
+#cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel6
+#cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel7
+#cp $basedir/index.php ./LegacyPlots/Legacy$year/$channel/$sel8
```

Comment:
- the should be uncommented
- should be fixed from `...$channel/$sel1` to `...$channel\_$tag/$sel1`